src/stacks/github/index.ts

```diff
@@ -0,0 +1,24 @@
+import { StackRegister } from '../stack-register';
+import * as util from 'util';
+import * as cp from 'child_process';
+
+@StackRegister.register
+export class GitHub {
+  constructor(readonly rootPath: string = './') {}
+
+  async isAvailable(): Promise<boolean> {
+    const exec = util.promisify(cp.exec);
+
+    return exec(`git -C ${this.rootPath} remote -v`)
+      .then((value: { stdout: string; stderr: string }) => {
```

Comment:
Do you need stderr ?
```suggestion
      .then(({ stdout: string }) => {
```